ssz/src/main/java/tech/pegasys/teku/ssz/backing/type/BasicViewType.java

```diff
@@ -59,4 +61,35 @@ public TreeNode createBackingNode(C newValue) {
   @Override
   public abstract TreeNode updateBackingNode(
       TreeNode srcNode, int internalIndex, ViewRead newValue);
+
+  private int getSSZBytesSize() {
+    return (getBitsSize() + 7) / 8;
+  }
+
+  @Override
+  public boolean isFixedSize() {
+    return true;
+  }
+
+  @Override
+  public int getFixedPartSize() {
+    return getSSZBytesSize();
+  }
+
+  @Override
+  public int getVariablePartSize(TreeNode node) {
+    return 0;
+  }
+
+  @Override
+  public int sszSerialize(TreeNode node, Consumer<Bytes> writer) {
+    Bytes ret = node.hashTreeRoot().slice(0, getSSZBytesSize());
+    writer.accept(ret);
+    return ret.size();
+  }
+
+  @Override
+  public TreeNode sszDeserialize(Bytes ssz) {
+    throw new UnsupportedOperationException("TODO");
+  }
```

Comment:
Should we just omit the `sszDeserialize` method for now until we actually implement it?